scripts/quests/otherAreas/An_Affable_Adamantking.lua

```diff
@@ -0,0 +1,321 @@
+-----------------------------------
+-- An Affable Adamantking
+-----------------------------------
+-- Log ID: 4, Quest ID: 107
+-- Beadeaux Zone     : !pos -45 24 60 147
+-- Peshi Yohnts      : !pos -6 -6 -145 241
+-- Beastmen's Banner : !pos 10 25 -50 148
+-----------------------------------
+-- !addquest 4 107
+-- !addItem 885 -- Turtle Shell
+-- !addItem 1637 -- Bugard Leather
+-- !addItem 15201 -- Quadav Barbut
+-----------------------------------
+require('scripts/globals/npc_util')
```

Comment:
This require is unnecessary, globals are preloaded on server start

Reply:
Removed